### ISSUE TITLE
Update org should not check email validation.

### ISSUE THREAD

**Issue by lunny** [go-gitea/gitea#13675]
```
2020/11/23 11:20:35 ...uters/org/setting.go:94:SettingsPost() [E] UpdateUser: e-mail invalid [email: ]
```

**Comment by 6543** [comment]
@lunny witch version did you test?, this should already be fixed.

**Comment by lunny** [comment]
I will update gitea.com and will close this if it's disappeared.

**Comment by 6543** [comment]
should be closed by #13627

**Comment by 6543** [comment]
@lunny pleace reopen if it is still not gone
